src/styles/blocks/about-us.scss

```diff
@@ -0,0 +1,70 @@
+.about-us {
+  display: flex;
+  flex-direction: column;
+  background: $color-bg-additional;
+  margin-bottom: 64px;
+  @include onTablet {
+    margin-bottom: 120px;
+  }
+
+  &__conteiner {
+    padding-top: 48px;
+    padding-bottom: 48px;
+
+    @include onTablet {
+      padding-top: 120px;
+      padding-bottom: 120px;
+    }
+
+    @include pageGrid;
+  }
+
+  &__background-vinil {
+    height: 200px;
+    width: 100%;
+    background-image: url(../images/vinil-bg.png);
+    background-size: cover;
+    background-position: center;
+
+    @include onTablet {
+      height: 400px;
+    }
+
+    @include onDesktop {
+      height: 460px;
+    }
+  }
+
+  &__title {
+
+    grid-column: 1 / -1;
+
+    @include onTablet {
+      grid-column: 1 / 4;
+    }
+
+    @include onDesktop {
+      grid-column: 1 / 7;
+    }
+  }
+
+  &__description {
+
+    font-size: 16px;
+    margin: 0;
```

Comment:
You can use reset file with reset styles instead of set the property for each element